documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -2788,11 +2788,15 @@ For example: `P1Y2M3DT4H5M6.78S`.
 For more information, see xref:postgresql-basic-types[PostgreSQL basic types].
 
 |[[postgresql-property-database-sslmode]]<<postgresql-property-database-sslmode, `+database.sslmode+`>>
-|`disable`
+|`prefer`
```

Comment:
Looking at those PRs for PG and MySQL.... instead of using the database's specific pass-through parameters and options... wouldn't it make more sense to provide a semantic option like `use.secure.connections` which can have 3 or more options, eg. `auto/prefer` --> tries to connect via secure connection but falls back to insecure when no secure connection can be established ; `enforce` --> tries secure connection and fails when secure connection cannot be established ; `disable` --> don't use secure connections at all ; `strict` --> added later to enforce cert validation ; 
Wdyt? @jpechane @Naros

Reply:
I filed https://issues.redhat.com/browse/DBZ-6397 for this